readme.md

```diff
@@ -66,6 +66,9 @@ module.exports = {
         repo: "github-repo",
         circleApi: "circleApiToken",
         googleJsonPath: "path/to/google/json",
+        storePassword: "password",
```

Comment:
These could be prefixed as keystore, so keystorePassword, keystoreAlias keystoreAliasPassword  or something like that so it is clear that they are related.